bigquery/deploy/README.md

```diff
@@ -55,14 +55,7 @@
 * Run indexer on GKE
   * If you are using the default [platinum_genomes dataset](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery/config/platinum_genomes),
 don't forget to [copy to your project and set project IDs in facet_fields.csv](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/bigquery#quickstart).
-  * Build, tag, and upload the docker image to GCR:
-    ```
-    cd bigquery
-    docker build -t gcr.io/PROJECT_ID/bq-indexer .
-    docker push gcr.io/PROJECT_ID/bq-indexer
-    ```
-  * Update `bigquery/deploy/bq-indexer.yaml` with the desired MY_GOOGLE_CLOUD_PROJECT and
-  EXTERNAL_IP.
+  * Update EXTERNAL_IP in `bigquery/deploy/bq-indexer.yaml`
```

Comment:
We should give users the choice.

  * By default, the indexer will be deployed using the latest image in the data-explorer-images project. If you want to use your own image:
```
 -    cd bigquery		
￼ -    docker build -t gcr.io/PROJECT_ID/bq-indexer .		
￼ -    docker push gcr.io/PROJECT_ID/bq-indexer
```
    And in `bq-indexer.yaml`, change `data-explorer-images` to your own project.